js/maps/mapTemplates.js

```diff
@@ -1,33 +1,36 @@
-var WL = app.TILES.WALL;
-var DRT = app.TILES.WALL;
-var G = app.TILES.GRASS;
+var W = app.TILES.WALL;   var H = app.TILES.HOUSE;
```

Comment:
I dunno where you learned this syntax `var a = 1; var b = 2;` there's nothing wrong with it, but it's generally not done by convention. But anyway not really important just pointing it out.